app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] The associations between models must be set up correctly according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). Great job so far 👏🏻 however let's specify the `author_id` column of the Post, Comment and Like Table as the  `foreign_key` since we're using the `author_id` column to reference the user modal.
🤔 Your code should look like this:

```ruby
  # app/models/user.rb
  has_many :posts, foreign_key: 'author_id'
  has_many :comments, foreign_key: 'author_id'
  has_many :likes, foreign_key: 'author_id'
```